lib/blockchain/cardano.js

```diff
@@ -0,0 +1,47 @@
+const fs = require('fs')
+const path = require('path')
+const common = require('./common')
+const mnemonicHelpers = require('../../lib/mnemonic-helpers')
+
+const coinUtils = require('../coin-utils')
+const options = require('../options')
+
+module.exports = { setup }
+
+const es = common.es
+
+// todo: maybe add '#' as second parameter?
+function escapeRegExp(str) {
+  return str.replace(/[.*+?^${}()|[\]\\#]/g, '\\$&')
+}
+
+function setup(dataDir) {
+  const coinRec = coinUtils.getCryptoCurrency('ADA')
```

Comment:
Is `coinRec` shortcut of `coinRecord`? :thinking: If yes, could you use the longer it might remove possible confusion when someone sees it below in the code.